frontend/src/components/MyCardList.tsx

```diff
@@ -1,39 +1,25 @@
 import MyCard from "./MyCard";
-import {useContext, useEffect, useState} from "react";
+import {useContext} from "react";
+import {Box} from "@mui/system";
 import TransactionContext from "../context/transaction/TransactionContext";
-import {toast} from "react-toastify";
-import TransactionDto from "../model/TransactionDto";
 
 export default function MyCardList() {
 
-    const {getAllTransactions, deleteTransaction} = useContext(TransactionContext);
-
-    const [allTransactions, setAllTransactions] = useState<TransactionDto[]>([]);
-
-    useEffect(() => {
-
-        getAllTransactions()
-            .then(data => {
-                return setAllTransactions(data)
-            })
-            .catch(error => {
-                toast.error(error.message)
-            });
-
-    }, [allTransactions])// eslint-disable-line
+    const {deleteTransaction, allTransaction} = useContext(TransactionContext);
 
     return (
-        <>
+        <Box>
+            <h1>Alle Transaktionen</h1>
+
             {
-                allTransactions.length !== 0 ? (
-                    allTransactions.map(t =>
-                        <MyCard allTransaction={t} deleteTransaction={deleteTransaction}/>
+                allTransaction.length !== 0 ? (
+                    allTransaction.map(t =>
+                        <MyCard key={t.id} allTransaction={t} deleteTransaction={deleteTransaction}/>
                     )
                 ) : (<h1>Keine Transaktion vorhanden</h1>)
```

Comment:
Only one h1 per page, maybe conditionally replace the heading text or use another element here. Maybe a p?